src/smif/convert/adaptor.py

```diff
@@ -8,12 +8,29 @@
 
 import numpy as np
 from smif.exception import SmifDataNotFoundError
+from smif.metadata import Spec
 from smif.model import Model
 
 
 class Adaptor(Model, metaclass=ABCMeta):
     """Abstract Adaptor, to convert inputs/outputs between other Models
+
+    Override method `generate_coefficients`, which accepts two
+    :class:`~smif.metadata.spec.Spec` definitions.
+
     """
+    @classmethod
```

Comment:
Instead of adding this here, we could put it on `smif.model.Model` and remove the implementation from `smif.model.SectorModel` - then we'll be left with `smif.model.Model.from_dict` as default and just the one override in `smif.model.ScenarioModel.from_dict`.